impl/client/src/main/javascript/web/pentaho/lang/Base.js

```diff
@@ -920,7 +920,14 @@ define([
     /* jshint validthis:true*/
 
     if(fun.is(value)) {
-      this[name] = methodOverride(value, this[name], rootProto, name);
+      // Only override if it is a normal property.
+      // When there is not baseDesc, methodOverride may need to provide a default base method.
+      var baseDesc = O.getPropertyDescriptor(this, name);
+      if(baseDesc && (baseDesc.get || baseDesc.set)) {
```

Comment:
No, this only applies when the _source_ property is *not* a getter/setter — which is how you'd override a getter/setter property —, but the target property is.